packages/darklang/stdlib/string.dark

```diff
@@ -28,9 +28,11 @@ module Darklang =
           else
             "a"
 
+
```

Comment:
Looking through the list functions, we shouldn't convert them to lists, operate, and then convert them back. There's huge overhead to that which is not possible to avoid at the moment.